### PR TITLE
feat(toast, gesture, divider): improved toast, added up/down gestures and vertical divider

### LINKED ISSUE #4583: Toast: fab positions
FAB position should only change when toast would otherwise overlay FAB. 

- swiping away a toast, it should not rotate
- worth noting in the text that on mobile it's full width and always at the bottom. 
- should be able to swipe in the direction the toast came from. So if the toast came from the bottom, you should be able to swipe down to dismiss

> doublecheck toast specks: https://spec.googleplex.com/quantum/components/snackbars-toasts.html#

screen shot 2015-09-11 at 9 03 09 am

### PR DESCRIPTION
**Pull request by EladBezalel**
* Added up and down gestures
* Added support in vertical divider
* Toast -
  * Fade away is not rotating anymore
  * When on mobile, toast is always full-width and on the bottom
  *  Now able to swipe in the direction the toast came from. So if the toast came from the bottom, you can swipe down to dismiss.

fixes #4583, #930 and #2228

### CODE REVIEW

**Comment by DevVersion** [comment]
I just want to notify you about the continuous integration error, perhaps you didn't see it :+1:

**Comment by EladBezalel** [comment]
This is because i run tests on Chrome locally and not on Phantom, will fix, thanks @DevVersion!

**Comment by ThomasBurleson** [comment]
@EladBezalel - please rebase.

File: src/components/divider/divider.scss [hunk 1]
```diff
@@ -8,3 +8,11 @@ md-divider {
     margin-left: $baseline-grid * 10;
   }
 }
+
+.layout-row {
+  & > md-divider {
```
Comment (ThomasBurleson) [inline src/components/divider/divider.scss]: Where is the demo associated with this?

**Pull request closed without merging**